src/styles/partials/components/_site-nav.scss

```diff
@@ -72,7 +72,7 @@ $mobile-menu-width: 245px;
 
         i {
           text-align: center;
-          margin-right: $margin-small;
+          margin-right: $margin-small + $base-spacing-unit;
```

Comment:
@sdurphy is asking for the default margin here, which is set in the variable `$default-margin`. Please use that variable instead.

Reply:
used default-margin